README.md

```diff
@@ -45,7 +51,24 @@ It will find the "real" IP in X-Forward headers and modify the request.info.remo
 
 ```js
 server.register([
-	require('therealyou'),
-	require('hapi-auth-ip-whitelist')
+	{
+		plugin: require('therealyou')
+	},
+	{
+		plugin: require('hapi-auth-ip-whitelist')
+	}
```

Comment:
What do you think of using format `{ plugin: require('therealyou') }` instead?

Reply:
Did for code consistency, to have all imported/required modules at the top of the file. But sure I can move it.

Reply:
Leave it, is fine!